bundles/net.tourbook.cloud/src/net/tourbook/cloud/suunto/SuuntoCloudDownloader.java

```diff
@@ -247,6 +252,8 @@ public void done(final IJobChangeEvent event) {
                         infoText);
                });
             }
+
+            PlatformUI.getWorkbench().getDisplay().asyncExec(() -> PersonManager.get_contributionItem_PersonSelector().setEnabled(true));
```

Comment:
I think that the enablement should be put into a `try {} finally {}` block to ensure it is enabled when an exception occurs

Reply:
Fixed

Reply:
I removed the `try {} finally {}` because I discovered, that the `done()` method in the job listener is always called also by failures

![image](https://github.com/mytourbook/mytourbook/assets/1283445/0a6ccdc0-1cb9-446e-8a75-27fd4510d4be)

Reply:
Correct, it is the final code called at the end of the Job